docs/operations/terraform_provisioning_azure_en.md

```diff
@@ -68,12 +77,18 @@ The following variables are available
 | aks_node_count          | The number of nodes to create                                         | 2             |
 | aks_vm_size             | The size of the Virtual Machine fo the nodes, such as Standard_DS2_v2 | Standard_B2s  |
 | ecosystem_namespace     | The namespace for the CES                                             | ecosystem     |
+| image_registry_url      | The endpoint for the docker-image-registry                            | -             |
 | image_registry_username | The username for the docker-image-registry                            | -             |
 | image_registry_password | The password for the docker-image-registry                            | -             |
-| image_registry_password | The email for the docker-image-registry                               | -             |
 | dogu_registry_username  | The username for the dogu-registry                                    | -             |
 | dogu_registry_password  | The password for the dogu-registry                                    | -             |
 | dogu_registry_endpoint  | The endpoint for the dogu-registry                                    | -             |
+| helm_registry_url       | The endpoint for the helm-registry                                    | -             |
+| helm_registry_username  | The username for the helm-registry                                    | -             |
+| helm_registry_password  | The password for the helm-registry                                    | -             |
+| setup_chart_version     | The chart version from k8s-ces-setup to install                       |               |
+| setup_chart_namespace   | The repository of the k8s-ces-setup chart                             |               |
+| ces_admin_password      | The CES admin password                                                | -             |
```

Comment:
```suggestion
```

Reply:
Why delete this? The variable is still available.